src/app/code/community/Meanbee/ConfigPoweredCss/Model/Config.php

```diff
@@ -77,6 +78,66 @@ public function getCssFilename($store = null)
      */
     public function getFullCssFilePath($store = null)
     {
-        return Mage::getBaseDir('skin') . '/frontend/base/default/' . $this->getCssFilename($store);
+        return $this->getCssDirectoryPath() . $this->getCssFilename($store);
+    }
+
+    /**
+     * Get directory for CSS files
+     *
+     * @param $store
+     * @return string
+     */
+    public function getCssDirectoryPath($store = null)
+    {
+
+        return Mage::getBaseDir('media') . $this->getCssPath($store);
```

Comment:
Needs a trailing slash after media directory `Mage::getBaseDir('media') . DS . $this->getCssPath($store);`